src/main/java/org/example/controllers/SurveyManagerController.java

```diff
@@ -161,13 +161,15 @@ public void addNumRangeQuestion(@RequestParam(value = "surveyID") Integer ID,
      * */
     @RequestMapping(value = "/closeSurvey", method = GET)
     @ResponseBody
-    public void closeSurvey(@RequestParam(value = "surveyID") Integer ID)
+    public String closeSurvey(@RequestParam(value = "surveyID") Integer ID)
```

Comment:
i think to keep our stuff consistent, we should leave the closeSurvey method for the JSON return, and create a new one called /surveyCloseSurvey for the templates

Reply:
Nevermind, misread the comment